site/app/templates/grading/simple/Display.twig

```diff
@@ -13,18 +13,7 @@
         </p>
     {% endif %}
 
-    <div class="column-wrapper" style="margin: 1em 0;">
-        <div id="checkpoint-sticky" class="column-wrapper">
-            {% include "grading/simple/StudentSearch.twig" %}
-            {% if action == 'lab' %}
-                <div>
-                    <p class="not-sticky">Undo/Redo Changes</p>
-                    <button id="checkpoint-undo" class="btn btn-primary key_to_click" tabindex="0" onclick="checkpointRollTo('{{ gradeable.getId() }}', -1)" disabled>Undo <i class="fas fa-undo"></i></button>
-                    <button id="checkpoint-redo" class="btn btn-primary key_to_click" tabindex="0" onclick="checkpointRollTo('{{ gradeable.getId() }}', 1)" disabled>Redo <i class="fas fa-redo"></i></button>
-                </div>
-            {% endif %}
-        </div>
-
+	<div class="column-wrapper" style="display: inline-block; width: 100%;">
```

Comment:
Should the inline CSS be moved to an external stylesheet?

Reply:
I would argue this isn't necessary since these styles aren't share for every column-wrapper though it is used by multiple.